src/OpenSSL/SSL.py

```diff
@@ -701,6 +711,35 @@ def set_default_verify_paths(self):
         """
         set_result = _lib.SSL_CTX_set_default_verify_paths(self._context)
         _openssl_assert(set_result == 1)
+        num = self._check_num_store_objects()
```

Comment:
You have to detect ``capath`` yourself. https://github.com/python/cpython/blob/master/Lib/ssl.py#L335  take the env vars into account. The four function return the hard-coded paths and the names of the env vars:

```
X509_get_default_cert_file_env()
X509_get_default_cert_file()
X509_get_default_cert_dir_env()
X509_get_default_cert_dir()
```

The names of the files in ``capath`` match the pattern ``[0-9a-f]{8}\.[0-9]``